client/src/pages/GrantAccessForm.js

```diff
@@ -27,15 +27,22 @@ const GrantAccessForm = () => {
 		try {
 			const res = await post(endpoint, data);
 
-			if (res && res.code !== "VALIDATION_ERROR") {
+			if (res.status === 403) {
```

Comment:
Did you test this? In your last message, you just told me that the catch block will be executed when we get a 403, if that's true, this will never be executed

Reply:
Yes. You are right. I have fixed it now.